source/tests/test_init_frz_model_multi.py

```diff
@@ -61,6 +61,7 @@ def _init_models():
     jdata = j_loader(str(tests_path / os.path.join("init_frz_model", "input.json")))
     fitting_config = jdata["model"].pop("fitting_net")
     loss_config = jdata.pop("loss")
+    learning_rate_config = jdata.pop("learning_rate")
```

Comment:
Is there still a breaking change? Old tests should be passed without modification.

Reply:
There is no breaking change now. I test it with water demo that using "learning_rate" in multi-task mode.